collections.go

```diff
@@ -40,7 +44,7 @@ func newCollectionsGetClient(sl *sling.Sling, id CollectionID) *CollectionsGetCl
 	return &CollectionsGetClient{sl: copy}
 }
 
-// Do makes the actual request for fetching said collection info.
+// Do make the actual request for fetching said collection info.
```

Comment:
Haha these are not typos. Your IDEs thinks `Do` is an  English auxiliary verb `do`  but here it is a noun so original comment is correct. The same goes for other `Do`s.